src/main/java/org/quiltmc/asmr/processor/capture/AsmrReferenceCaptureComparator.java

```diff
@@ -0,0 +1,38 @@
+package org.quiltmc.asmr.processor.capture;
+
+import java.util.Comparator;
+
+public class AsmrReferenceCaptureComparator implements Comparator<AsmrReferenceCapture> {
```

Comment:
What's the point of this class, if you're just using the lambda inside `AsmrProcessorRunner`? It's one or the other. Should probably also be given a more descriptive name that refers to the tree order. Also needs an `@ApiStatus.Internal` annotation to match `AsmrReferenceCapture` itself.

Reply:
I moved it out in a separate class to reduce the linecount, but with the merge, the lambda got readded and I had forgotten that I moved that part out, so I just quickly recoded the lambda again.

Reply:
Ah my bad